.travis.yml

```diff
@@ -4,8 +4,8 @@ language: node_js
 sudo: false
 
 node_js:
-  - "4.2"
```

Comment:
Doesn't it? https://github.com/felixrieseberg/ember-electron/blame/feat/forge/README.md#L16